java/core/src/java/org/apache/orc/impl/RunLengthIntegerWriterV2.java

```diff
@@ -545,14 +547,10 @@ private void determineEncoding() {
 
   private void computeZigZagLiterals() {
     // populate zigzag encoded literals
-    long zzEncVal = 0;
-    for (int i = 0; i < numLiterals; i++) {
-      if (signed) {
-        zzEncVal = utils.zigzagEncode(literals[i]);
-      } else {
-        zzEncVal = literals[i];
+    if (signed) {
```

Comment:
Shall we remove `if` statement and add an assertion at the first line of this function?

Reply:
I quite agree. fix in 7de084b